library.json

```diff
@@ -24,6 +24,9 @@
     {
       "name": "AsyncTCP",
       "platforms": "espressif32"
+    },
+    {
+      "name": "Hash"
```

Comment:
this needs to have ```"platforms" : "espressif"``` as this dependency is only used on the esp8266.

Reply:
Yes, you're right. I saw in the code the #ifndef. Fixed.